sources/api/schnauzer/src/helpers.rs

```diff
@@ -98,6 +101,9 @@ const KUBE_RESERVE_3_CORES: f32 = KUBE_RESERVE_2_CORES + 5.0;
 const KUBE_RESERVE_4_CORES: f32 = KUBE_RESERVE_3_CORES + 5.0;
 const KUBE_RESERVE_ADDITIONAL: f32 = 2.5;
 
+const IPV4_LOCALHOST: &str = "127.0.0.1";
+const IPV6_LOCALHOST: &str = "::1";
```

Comment:
You can make these `IpAddr` constants to avoid doing the conversion at runtime:
```suggestion
const IPV4_LOCALHOST: IpAddr = IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1));
const IPV6_LOCALHOST: IpAddr = IpAddr::V6(Ipv6Addr::new(0, 0, 0, 0, 0, 0, 0, 1));
```